metaflow/plugins/aws/awslambda/lambda_runner.py

```diff
@@ -0,0 +1,314 @@
+import shlex
+from threading import Thread
+import boto3
+import json
+from typing import Dict, Optional, TYPE_CHECKING, Callable
```

Comment:
It would be great to find a way to make AWS Lambda compatible with Py2 as well if it's not too much work.